docker/include/bin/start_faux

```diff
@@ -161,17 +161,17 @@ fi
 if [ -n "${options[ntpv4]}" ]; then
     dhcp_ntp=$(fgrep NTPSERVERS= /run/ntpdate.dhcp)
     ntp_server=`echo $dhcp_ntp | cut -d "'" -f 2`
-    (for i in {1..10}; do 
+    (for i in {1..15}; do 
         echo Transmitting NTP query to $ntp_server using NTPv4
         java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar $ntp_server 123 4 > ntp.log &
```

Comment:
Just a Nit/Cleanup: NTPClient has a timerPeriod and the option to send requests multiple times. Should the timer and iterations be parameters to it, instead of looping here? (Alternately, since we are using NTPClient as a one-shot, remove the timerPeriod option from it to avoid confusion around its use?)